WordPress/Classes/ViewRelated/NUX/SigninLinkRequestViewController.swift

```diff
@@ -35,6 +35,8 @@ class SigninLinkRequestViewController: NUXAbstractViewController {
         if !email.isValidEmail() {
             assert(email.isValidEmail(), "The value of loginFields.username was not a valid email address.")
         }
+        
+        self.sourceTag = SupportSourceTag.wpComLogin
```

Comment:
Would it make sense to specify this as part of the wpcom magic link flow vs the default wpcom screenname / password flow?